garrysmod/gamemodes/terrortown/gamemode/radar.lua

```diff
@@ -17,7 +17,11 @@ local function RadarScan(ply, cmd, args)
 
          ply.radar_charge =  CurTime() + chargetime
 
-         local scan_ents = player.GetAll()
+         local scan_ents = {}
+         for i, pl in player.Iterator() do
+            scan_ents[i] = pl
+         end

```

Comment:
If this was faster than calling `player.GetAll`, then wouldn't `player.GetAll` just be reimplemented this way?

Regardless, this function does not get called very frequently so the simpler implementation of just calling `player.GetAll()` is preferable here.

Reply:
player.GetAll returns a table, not an iterator, so it can't be reimplemented.